native-messaging/app/ping_pong.json

```
@@ -1,7 +1,7 @@
 {
   "name": "ping_pong",
   "description": "Example host for native messaging",
-  "path": "/path/to/native-messaging/app/ping_pong.py",
+  "path": "/Users/svincent/dev/webextensions-examples/native-messaging/app/ping_pong.py",
```

```suggestion
  "path": "/path/to/native-messaging/app/ping_pong.py",
```

Let's keep the obvious dummy path instead of your local filesystem directory structure.

---

Oops, didn't mean to commit that. Thanks for catching!